src/styles/products.scss

```diff
@@ -0,0 +1,23 @@
+$width: calc((200px * 4) + (48px * 3));
```

Comment:
what if width of card changes, or their number, or a gap? store that values in variables too